Images_And_Graphics_style_v1.css

```diff
@@ -333,7 +372,12 @@ ul li:hover ul {
 }
 
 .logo1 {
-	float: right;
+	float: left;

```

Comment:
I changed float: right to float: left as I didn't like where the image logo was placed so I placed it next to the text but on the left side so the other images can look good together.